content_handler/vulkan_surface.h

```diff
@@ -43,18 +44,17 @@ class VulkanSurface : public flow::SceneUpdateContext::SurfaceProducerSurface,
       std::function<void(void)> on_writes_committed) override;
 
   // |flow::SceneUpdateContext::SurfaceProducerSurface|
-  uint32_t GetSessionImageID() const override;
+  mozart::client::Image* image() override;
```

Comment:
The interface is defined in the `flow` library and there used to be two implementations in the content handler (one for Vulkan and one for software rasterization). The `flow` layer not know anything about the specific client rendering library used. Hence the virtual methods. I'll fix the signature.